content/en/docs/concepts/security/service-accounts.md

```diff
@@ -196,6 +196,31 @@ or using a custom mechanism such as an [authentication webhook](/docs/reference/
 You can also use TokenRequest to obtain short-lived tokens for your external application.
 {{< /note >}}
 
+### Enforcing Mountable Secrets using Annotations {#enforce-mountable-secrets}
+
+Kubernetes provides an annotation called `kubernetes.io/enforce-mountable-secrets` that you can add to your ServiceAccounts.
+When this annotation is applied, the service account's secrets can only be mounted on specified types of resources, enhancing the security posture of your cluster.
+
+#### How to Add the Annotation
+
+You can add the annotation to a ServiceAccount using a manifest:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    kubernetes.io/enforce-mountable-secrets: "true"
+  name: my-serviceaccount
+  namespace: my-namespace
+```
+
+#### Effect on the Cluster
+
+When this annotation is set to true, the Kubernetes control plane ensures that the secrets belonging to this ServiceAccount are only mountable on the resource types specified in the annotation value.
```

Comment:
```suggestion
When this annotation is set to "true", the Kubernetes control plane ensures that
the Secrets from this ServiceAccount are only mountable on the resource kinds
specified in the annotation value.
```

Reply:
This looks wrong. How confident are you @chansuke that this is the actual behavior of Kubernetes?